src/cljs/dato/lib/core.cljs

```diff
@@ -40,31 +41,26 @@
 (defn bootstrapped? [db]
   (dsu/qe-by db :dato.meta/bootrapped? true))
 
-(defn new-dato [dato-host dato-path app-db cs-schema]
+(defmulti ws-message-handler (fn [dato-ch msg] (:op msg)))
```

Comment:
Can we get a line of whitespace between every definition? Eventually we'll get something like clj-fmt for cljs in place, but for now would like to lean towards a standard coding convention.